docs/argo-server-sso-argocd.md

```diff
@@ -107,6 +107,44 @@ data:
     redirectUrl: https://argo-workflows.mydomain.com/oauth2/callback
 ```
 
+## Example SSO config with Secrets from environment variable
+
+The following example shows how you can use environment variables to get your secrets. This is
+especially useful if you use CSI driver to get your secrets from, say, Azure Key Vault.
```

Comment:
Are you referencing the [Secrets Store CSI Driver](https://secrets-store-csi-driver.sigs.k8s.io/)? That does [_require_ that secrets are created in k8s first](https://secrets-store-csi-driver.sigs.k8s.io/topics/set-as-env-var) before you can use env vars. It's a CSI Driver, so the default is to _mount_ the secret, not set an env variable

Also in general, should be specific with what is being referenced in docs and avoid ambiguity